lib/config.py

```diff
@@ -297,6 +297,21 @@ def validate_config(CONFIG: CONFIG_DICT_TYPE) -> None:
     config_assert(CONFIG["challenge"]["preference"] in ["none", "human", "bot"],
                   "challenge.preference should be `none`, `human`, or `bot`.")
 
+    min_max_template = "challenge.max_{setting} < challenge.min_{setting} will result in no challenges being accepted."
```

Comment:
I think we should specify that no `real time` or `correspondence` challenges will be created, because if `max_base < min_base` then the bot could still accept correspondence challenges.